config/webpack/production.js

```diff
@@ -15,5 +16,21 @@ module.exports = merge(sharedConfig, {
     new CompressionPlugin({
       test: /\.(js|css|html|json|ico|svg|eot|otf|ttf)$/,
     }),
+
+    // Write out dependencies list for audit purposes
+    new StatsWriterPlugin({
+      filename: "webpack-modules-manifest.json",
+      fields: ["modules"],
+
+      transform(data) {
+        // recursively flatten nested modules
+        const transformModule  = m  => m.modules ? transformModules(m.modules) : m.name;
+        const transformModules = ms => ms.flatMap(m => transformModule(m));
```

Comment:
One detail... https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap#Browser_compatibility

We may need to update the minimum node version to 11 (package.json:172)

Reply:
What node version are we on?

Reply:
Oh I see 10... as I'd like to have this on jansa I assume we don't want to bump it... I could just add in a .map.reduce version if that's preferable.

Reply:
Me, randomly on 10.19.0 :)

@simaishi would know what we're actually building with.

Reply:
We're on 12.x for master/jansa

Reply:
@himdel What is your preferred way forward?  Bump to 12, or keep it at 10 and I do it differently?

Reply:
I'd go with a bump, if we're already using node 12.

Reply:
(That said, if we're going to bump to node 12, I need someone to approve https://github.com/ManageIQ/manageiq-ui-classic/pull/7098, so that people switching between master and ivanchuk have a chance :).)

Reply:
Ok updated to Node 12.